### ISSUE TITLE
Can't find fields

### ISSUE THREAD

**Issue by KatieMFritz** [Nav33d/craft-fieldreveal#1]
Hi there! Just installed this plugin for a site using Craft Pro 3.1.26, PostgreSQL 10.8.

I'm getting a 500 error for `/actions/fieldreveal/default/get-fields` that's breaking the plugin for me.

The error is: 
```
Database Exception – yii\db\Exception
SQLSTATE[42703]: Undefined column: 7 ERROR: column fields.groupid does not exist
LINE 3: INNER JOIN "fields" "fields" ON fieldgroups.id = fields.grou...
^
HINT: Perhaps you meant to reference the column "fields.groupId".
The SQL being executed was: SELECT "fields"."id" AS "id", "fields"."name" AS "name", "fields"."handle" AS "handle", "fields"."type" AS "type", "fieldgroups"."name" AS "groupName"
FROM "fieldgroups" "fieldgroups"
INNER JOIN "fields" "fields" ON fieldgroups.id = fields.groupId
Error Info: Array
(
    [0] => 42703
    [1] => 7
    [2] => ERROR:  column fields.groupid does not exist
LINE 3: INNER JOIN "fields" "fields" ON fieldgroups.id = fields.grou...
                                                         ^
HINT:  Perhaps you meant to reference the column "fields.groupId".
)
↵
Caused by: PDOException
SQLSTATE[42703]: Undefined column: 7 ERROR: column fields.groupid does not exist
LINE 3: INNER JOIN "fields" "fields" ON fieldgroups.id = fields.grou...
^
HINT: Perhaps you meant to reference the column "fields.groupId".
in /Users/fritzk/code/facadetectonics/vendor/yiisoft/yii2/db/Command.php at line 1290
```

**Comment by Nav33d** [comment]
I will give it a test using PostgreSQL. It seems to work fine with MySQL. Could you please let me know if you are using any table prefix?

**Comment by KatieMFritz** [comment]
Thank you! We are not using a table prefix.

**Comment by Nav33d** [comment]
@KatieMFritz Apologies for the late response. I have just released version 1.0.1 which should fix this issue.
